modules/translation/translation.go

```diff
@@ -99,3 +100,66 @@ func (l *locale) Language() string {
 func (l *locale) Tr(format string, args ...interface{}) string {
 	return i18n.Tr(l.Lang, format, args...)
 }
+
+// Language specific rules for translating plural texts
+var trNLangRules = map[string]func(int64) int{
+	"en-US": func(cnt int64) int {
```

Comment:
de-DE is missing and it's same as "en-US"

Reply:
It's default to `en-US` for most languages, so there won't be a problem (and no duplicated code)

<img width="480" alt="image" src="https://user-images.githubusercontent.com/2114189/147864888-b9067c3c-f77a-46d6-b90f-4787da37883a.png">

Reply:
I added a comment here: `// the default rule is "en-US" if a language isn't listed here`, it's clear to everyone now 😊